backend/app/tests/api/routes/documents/test_route_document_upload.py

```
@@ -154,12 +154,12 @@ def test_upload_without_transformation(
     @patch("app.services.doctransform.job.start_job")
     def test_upload_with_transformation(
         self,
-        mock_start_job,
+        mock_start_job: Any,
```

Ruff (0.14.10)

Undefined name `Any`

(F821)